staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -77,8 +77,6 @@ type Event struct {
 	// AuditLevel at which event was generated
 	Level Level
 
-	// Time the request reached the apiserver.
-	Timestamp metav1.Time
```

Comment:
A agree with @crassirostris that we should have different `CreationTimestamp`s. In my mental model those events are distinct and the reuse in the handler chain is implementation detail.